### PR TITLE
Fix ansible-test relative import analysis.

### LINKED ISSUE #110: module_util changes don't trigger 'ci_complete'
##### SUMMARY

It used to be the case that if we made a change to module_utils it would trigger a ci_complete run.  This isn't happening today

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

CI

##### ANSIBLE VERSION

n/a

##### CONFIGURATION

n/a

##### OS / ENVIRONMENT

CI

##### STEPS TO REPRODUCE

Push a change to something in plugins/module_utils

##### EXPECTED RESULTS

All CI tests are automatically triggered

##### ACTUAL RESULTS

They're not :(

For example: https://app.shippable.com/github/ansible-collections/amazon.aws/runs/563/summary/console

### PR DESCRIPTION
**Pull request by mattclay**
##### SUMMARY

Fix ansible-test relative import analysis.

Replaces https://github.com/ansible/ansible/pull/70866
Resolves https://github.com/ansible-collections/amazon.aws/issues/110

##### ISSUE TYPE

Bugfix Pull Request

##### COMPONENT NAME

ansible-test

### CODE REVIEW

**Comment by mattclay** [comment]
cc @tremble

File: test/lib/ansible_test/_internal/import_analysis.py [hunk 1]
```diff
@@ -229,6 +259,33 @@ def __init__(self, path, module_utils):
...
+                    self.module = path_to_module(revised_path)
+                    break
+        else:
+            # This assumes that all files within the collection are executed by Ansible as part of the collection.
+            # While that will usually be true, there are exceptions which will result in this resolution being incorrect.
+            self.module = path_to_module(os.path.join(data_context().content.collection.directory, self.path))
+
     # noinspection PyPep8Naming
     # pylint: disable=locally-disabled, invalid-name
     def visit_Import(self, node):
```
Comment (tremble) [inline test/lib/ansible_test/_internal/import_analysis.py:291]: Dumb question: should visit_Import be updated too?

Comment (mattclay) [reply test/lib/ansible_test/_internal/import_analysis.py:291]: Only the `from ... import` statement supports relative imports, so there's nothing to do for the `import` statement.

**Review by tremble — APPROVED** [review]
For what it's worth I've tested this locally and it appears to be working as expected...

**Pull request merged**